code/game/gamemodes/objective.dm

```diff
@@ -421,6 +421,55 @@ GLOBAL_LIST(admin_objective_list) //Prefilled admin assignable objective list
 			return ..()
 	return SSshuttle.emergency.is_hijacked() || ..()
 
+/datum/objective/gimmick
+	name = "gimmick"
+	martyr_compatible = TRUE
+
+/datum/objective/gimmick/update_explanation_text()
+	var/selected_department = pick(list( //Select a department for department-based objectives
+		DEPT_SCIENCE,
+		DEPT_ENGINEERING,
+		DEPT_SECURITY,
+		DEPT_MEDICAL,
+		DEPT_SERVICE,
+		DEPT_SUPPLY,
+		DEPT_COMMAND
+	))
+	var/list/gimmick_list = list( //This list is ONLY for objectives that depend on a random selection
+		"Get everyone to hate [selected_department], through misdirection, slander, and sabotage.",
+		"Incite a strike in [selected_department].",
+		"You are a bully. Be a real jerk to those nerds at [selected_department] whenever possible. NEEERDS!",
+		"Keep power & cameras cut in [selected_department].",
+		"Disguise yourself as a member of [selected_department] and sabotage them from within.",
+		"Vandalize [selected_department] as much as possible.",
+		"Destroy [selected_department] utterly and entirely.", //Teeeeechnically, there is a 1/50 chance of getting any specific gimmick objective, making this one rarer than hijack
+		"Steal equipment critical for [selected_department] to function.",
+		"Attempt to kidnap a member of [selected_department] to take with you alive on an escape pod or a shuttle, for questioning at Syndicate HQ.",
+		"Hack all doors in [selected_department] to be shocked, bolted, or inaccessible by the AI"
+	)
+	if(target && target.current) //Only add these objectives to the pool if there are targets on the station
+		gimmick_list.Add(
+			"Construct an elaborate trap for [target.name].",
+			"Ensure [target.name] is demoted from their position.",
+			"Get [target.name] arrested for a crime they haven't committed.",
+			"Top-secret Syndicate intelligence suggests that [target.name] is responsible for scratching the paint on one of our CEOs' private cruisers. Make their life as miserable as possible without killing them.",
+			"Kidnap [target.name] and hold them for ransom!",
+			"Replace [target.name] completely.",
```

Comment:
seems redundant with changelings, and often ends up boring

Reply:
these objectives are for tators specfically, meaning they don't get the same shapeshifting abilities as changelings. this would require more effort from the tator to look and act the part, and maybe lead to interesting/funny scenarios considering they cant change their genetic makeup on a whim.

Reply:
@zeskorion 
The gimmicks are just there for inspiration, there isn't a check for them and its just for ideas and to have some fun.
You can ignore it as always, I say leave it in this is a fun idea.